source/api/auth/0.9/index.md

```diff
@@ -93,6 +93,9 @@ A browser running JavaScript retrieved from one domain cannot use `XMLHttpReques
 
 The server on the Resource Domain treats the access token as a representation of, or proxy for, the cookie that gains access to the Content Resources. When the client makes requests for the Description Resources and presents the access token, the responses tell the client what will happen when the browser requests the corresponding content resources with the access cookie the access token represents. These responses let the client decide what user interface and/or Content Resources to show to the user.
 
+### 1.4. Security
+
+The purpose of this specification to support access-control for IIIF resources and hence security is a core concern. This specification prescribes the use of cookies and bearer tokens to convey authentication status. To prevent misuse, both cookies and bearer tokens need to be protected from disclosure in storage and in transport. Implementations _SHOULD_ use [HTTP over TLS][rfc-2818], commonly known as HTTPS, for all communication. All references to HTTP should be read assuming the use of HTTP over TLS. See also the [Implementation Notes][a-implementation-notes].
```

Comment:
I think you could strike this sentence: "This specification prescribes the use of cookies and bearer tokens to convey authentication status." and Just say "To prevent misuse, the cookies and bearer tokens described in this specification need to be protected ..."